core/sts-3.3.md

```diff
@@ -125,6 +195,27 @@ client to the server until the policy expires. At that time, the client will
 revert back to a non-secure connection. Servers SHOULD advertise a long enough
 duration which makes this scenario less likely to happen.
 
+Servers should choose an appropriate `duration` value according with reference to
+the "Server implementation considerations" section to avoid inadvertent expiry issues.
+
+### Denial of Service
+
+STS could be used to inflict Denial of Service (DoS) on IRC servers in a number of ways.
+Some non-exhaustive examples include:
+
+* An attacker can inject an STS policy into an insecure connection. This causes the
+client to reconnect on a secure port, which may fail persistently if the server isn't
+listening for secure connections. This can be mitigated in clients by allowing for
+STS policy rejection as described in the "Client implementation considerations" section.
```

Comment:
Note that this can only happen persistently if the attacker is present at all times and is actively stripping the redirect, it will not be persistent otherwise.